modules/tile-converter/src/3d-tiles-converter/helpers/load-i3s.ts

```diff
@@ -40,13 +49,83 @@ export const loadI3SContent = async (
       },
       _tilesetOptions: {
         store: sourceTileset.store,
+        // @ts-expect-error
         attributeStorageInfo: sourceTileset.attributeStorageInfo,
+        // @ts-expect-error
         fields: sourceTileset.fields
       }
     }
   };
-  const tileContent = await load(sourceTile.contentUrl, I3SLoader, loadOptions);
+  const tileContent = await loadFromArchive(
+    sourceTile.contentUrl,
+    I3SLoader,
+    loadOptions,
+    slpkFilesystem
+  );
 
-  // @ts-expect-error
   return tileContent;
 };
+
+/**
+ * Load local SLPK file to ZipFileSystem instance
+ * @param url - path to SLPK file
+ * @returns instance of ZipFileSystem or null if url is not an SLPK file
+ */
+export async function openSlpk(url: string): Promise<ZipFileSystem | null> {
```

Comment:
Should we capitalize consistently? 
```suggestion
export async function openSLPK(url: string): Promise<ZipFileSystem | null> {
```